lib/policies/serializationPolicy.ts

```diff
@@ -30,50 +31,45 @@ export class SerializationPolicy extends BaseRequestPolicy {
   }
 
   public async sendRequest(request: WebResource): Promise<HttpOperationResponse> {
```

Comment:
If you're removing the await call, then you should make this function non-async.

Reply:
Good catch